doc/develop/test/ztest.rst

```diff
@@ -489,6 +489,11 @@ following in your source::
 
     #include <zephyr/fff.h>
 
+Zephyr provides several FFF-based fake drivers which can be used as either stubs or mocks. Fake
+driver instances are configured via :ref:`devicetree` and :ref:`kconfig`. See the following
+devicetree bindings for more information:
+
+ - :dtcompatible:`zephyr,fake-can`
```

Comment:
To avoid bitrot: I'd suggest saying here something like
```rst
look for ``zephyr,fake-*``
```

Reply:
That's nice too, but having a direct reference makes it easier to find the relevant compatibles.

I am pondering creating a separate documentation page under testing, which describes the various fake drivers, emulated drivers, and simulators available in Zephyr - their design patterns and suggested use-cases. I'd like to handle that outside this PR, though.